### PR TITLE
Trigger PNET_EVENT_PRMEND after DControl response have been sent

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

File: src/device/pf_cmdev.c [hunk 1]
```diff
@@ -4869,7 +4869,8 @@ int pf_cmdev_rm_dcontrol_ind (
    pnet_t * net,
    pf_ar_t * p_ar,
    pf_control_block_t * p_control_io,
-   pnet_result_t * p_dcontrol_result)
+   pnet_result_t * p_dcontrol_result,
+   bool * p_set_state_paramend)
```
Comment (olbjo) [inline src/device/pf_cmdev.c:4873]: paramend too similar to PRMEND? use param_end or parameter_end or prmend

File: sample_app/sampleapp_common.c [hunk 2]
```diff
@@ -825,7 +825,15 @@ static int app_state_ind (
       (void)pnet_set_provider_state (net, true);
 
       /* Send application ready  */
-      os_event_set (p_appdata->main_events, APP_EVENT_READY_FOR_DATA);
+      if (p_appdata->arguments.verbosity > 0)
```
Comment (olbjo) [inline sample_app/sampleapp_common.c:828]: Maybe the old event shall be kept? 
An application that needs additional time to prepare before sending the application ready can then use the existing event to implement this.  This is a comment. I am not sure what is the best solution

**Review by olbjo — APPROVED** [review]

File: sample_app/sampleapp_common.c (hunk 2 above)
Comment (pyhys) [reply sample_app/sampleapp_common.c:828]: Ok. 
Done.

File: src/device/pf_cmdev.c (hunk 1 above)
Comment (pyhys) [reply src/device/pf_cmdev.c:4873]: Now `p_set_state_prmend`
Done

**Pull request merged**